android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -35,6 +44,57 @@ class RNPlayerView(context: ReactApplicationContext) : LinearLayout(context) {
             playerView?.player = value
         }
 
+    /**
+     * Object that handles PiP mode changes in React Native.
+     */
+    var pictureInPictureHandler: RNPictureInPictureHandler? = null
+
+    /**
+     * Whether this view should pause video playback when activity's onPause is called.
+     * By default, `shouldPausePlayback` is set to false when entering PiP mode.
+     */
+    private var shouldPausePlayback: Boolean = true
```

Comment:
The name is very generic, a bit miss leading as we do not pause the player directly and I think it does not provide a good idea on when the player should be pause.

Something like the following might give a bit more context.

```
    private var shouldPausePlaybackOnActivityPause: Boolean = true
```
Feel free to come up with an own variable name :) 

I like that you started with `should` 👍

Reply:
Alright, that makes sense. I'll rename it to what you suggested, then. This name seems good enough 🙂.

Reply:
Done 9108c08059e126de8ff5e8da1404a4092c63bba7